cmd/api/config.go

```diff
@@ -115,11 +112,6 @@ type ChainConfig struct {
 		BlockFailedExecutionBackoff string `default:"10s"`
 		DedupExecutedTxns           bool   `default:"false"`
 	}
-	NonceTracker struct {
```

Comment:
the nonce tracker was the module that was keeping track of balances, we're not going to start it anymore. so we don't need these configs